static-site/content/blog/2024-12-10-auspice-2024.md

```diff
@@ -0,0 +1,121 @@
+---
+author: "The Nextstrain team"
+date: "2024-12-10"
+title: "Notable changes in Auspice"
+sidebarTitle: "Notable changes in Auspice"
+---
+
+The Nextstrain team continues to improve the visualisation capabilities of
+Auspice, including new features, bug fixes and performance improvements. This
+year, we've released 19 versions and wanted to highlight some of the
+significant changes.
+
+<!--
+number of git tags in 2024 (generated by GitHub Copilot):
+git tag --list --format='%(creatordate:short)' | grep '^2024' | wc -l
+-->
+
+## Table of contents
+
+- [Focus on Selected](#focus-on-selected)
+- [View in Other Platforms](#view-in-other-platforms)
+- [Performance improvements](#performance-improvements)
+- [Conveying confidence of tips through saturation (version 2.56)](#conveying-confidence-of-tips-through-saturation-version-256)
+- [Filtering on all metadata](#filtering-on-all-metadata)
+- [Full changelog](#full-changelog)
+
+## Focus on Selected
+
+When viewing large phylogenetic trees with a sparse filter, it can be difficult
+to really see and understand the selected tips as they occupy a minority of the
+vertical space available. Enabling **Focus on Selected** allows the selected
+tips to take up more vertical space compared to other tips. For example,
+[filtering the SARS-CoV-2 global 6m view to one of the earliest clades, 20I
+(Alpha)](https://nextstrain.org/ncov/gisaid/global/6m@2024-12-10?d=tree&f_clade_membership=20I%20%28Alpha%29&p=full):
+
+[![Focus on Selected example](/blog/img/focus-on-selected-example.png)]()
+
+You can toggle this focus on/off in the sidebar:
+
+[![Focus on Selected toggle](/blog/img/focus-on-selected-toggle.png)]()
+
+We [plan](https://github.com/nextstrain/auspice/issues/1368) to continue
+exploring better ways to understand sets of sequences in the context of the
+larger tree, initially by better use of the horizontal axis which should help
+for early clades in the tree such as 20I (Alpha).
+
+## View in Other Platforms
+
+Interoperability between platforms is something we've been big proponents of,
+and are part of [PH4GE](https://pha4ge.org/) to help support this. For many
```

Comment:
As per comment in the google doc, can someone double check this

Reply:
Independent of James's comment: during the move from the GDoc to here, this got turned from a parenthetical aside to a secondary clause in the sentence. The sentence flow is very awkward as a result -- the verb tenses are muddled ("have been" vs "are" ) and the "this" reference reaches a long way back for its antecedent. 

I would advocate for changing this to something like "Historically, we have always been strong advocates for interoperability between platforms." and leave it at that, unless there's a specific reason to bring PH4GE into it.

Reply:
Thanks, I like your suggested sentence. Switched to it:

https://github.com/nextstrain/nextstrain.org/blob/58e0190f82bd5575043b66253378bf2579c87289/static-site/content/blog/2024-12-10-auspice-2024.md#L49-L50

Also, I've split the changes into verbatim GDoc conversion (706eb89d11d033ce2e183bdf6bb91541da6ad91b) and my edits on top of that (ca88c2e9484fd48611ebef3778f889144863eb98) to solicit feedback on changes such as this one.